code/modules/projectiles/guns/specialist.dm

```diff
@@ -489,71 +489,45 @@
 //-------------------------------------------------------
 //GRENADE LAUNCHER
 
-/obj/item/weapon/gun/launcher/m92
-	name = "\improper T-26 grenade launcher"
-	desc = "A heavy, 6-shot grenade launcher used by the TerraGov Marine Corps for area denial and big explosions."
-	icon_state = "m92"
-	item_state = "m92"
-	max_shells = 6 //codex
-	caliber = CALIBER_40MM //codex
-	load_method = SINGLE_CASING //codex
+/obj/item/weapon/gun/launcher
```

Comment:
Rocket launchers share this subtype. You should probably make rockets there own subtype from what GLs are.